src/main/java/codeu/controller/SearchServlet.java

```diff
@@ -0,0 +1,91 @@
+package codeu.controller;
```

Comment:
Tests for this servlet?

Reply:
Will add the tests in a future PR, sorry I forgot to mention that.